ferveo/BENCHMARK.md

```diff
@@ -0,0 +1,79 @@
+# WASM results
```

Comment:
What's the purpose of this file as part of the codebase? Is it expected to change regularly?

Reply:
I expect it to change at least once after we finalize changes to the DKG scheme (simple vs fast decryption). We also introduced [other changes](https://github.com/nucypher/ferveo/pull/14) that I want to keep track of.